### ISSUE TITLE
Ribbons and ref arrows scale to the size of the slot the Logix interface belongs to

### ISSUE THREAD

**Issue by shiftyscales** [Frooxius/NeosPublic#529]
Steps to replicate:
- Create a Logix interface on a slot that is scaled to any non-1,1,1 size.
- Connect Logix nodes/drive arrows to the interface

Observations:
- This occurs for Logix interfaces for the slot, as well as interfaces for components of the slot

https://cdn.discordapp.com/attachments/598212542938218506/709513842690097213/2020-05-11_14-11-50.mp4

**Comment by Frooxius** [comment]
This is technically correct behavior, that's how the system was originally designed. It's not a desirable behavior, but it needs the system to be redesigned, it's not something that that can be just "fixed".

**Comment by shiftyscales** [comment]
I figured as much. And wasn't sure which tag is best for "is technically correct, but undesirable"- I'll try to use 'enhancement' in the future for such cases, and reserve 'bug' for technical issues then. Thanks!

Also just wanted to document this behavior in an issue too.

**Comment by TehTurk** [comment]
(Finger.on mobile >:[) Would it be possible to have the scale of the ref be relative to the tooltip itself?

**Comment by Anomalous** [comment]
As long as we're documenting technically correct but undesirable behaviors with logix ref arrows, I want to add that reference arrows disappearing when they reference inactive slots is undesirable. It's pretty obvious why once you realize that ref arrows get parented under the slot they're referencing. However, it's confusing at first, and even once you know what's going on it means the arrows can't be moved until you enable the referenced slot (or re-parent the ref arrow).

**Comment by Frooxius** [comment]
Yes, that's pretty much part of the same problem as well. Since they get parented under the slot, they get disabled with it, there's not much way around that, since it's not possible to have active object under inactive hierarchy.

With the redesign I want to decouple the LogiX interfaces and references from being too "physical" with the stuff they reference, which would get rid of a lot of these problems, including this one.
